pygotham/frontend/sponsors.py

```diff
@@ -73,7 +73,7 @@ def edit(pk):
     return render_template('sponsors/edit.html', form=form, sponsor=sponsor)
 
 
-@route(blueprint, '/')
+@route(blueprint, '/', navbar_kwargs={'path': ('Sponsors', 'Sponsors')})
```

Comment:
Can this be reduced to just the Sponsors top nav item as per our discussions with Allison?